spec/integ/crypto/verification.spec.ts

```diff
@@ -1259,14 +1259,17 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
 
             const requestId = await requestPromises.get("m.megolm_backup.v1");
 
+            const keyBackupIsCached = new Promise<void>((resolve) => {
+                aliceClient.on(CryptoEvent.KeyBackupDecryptionKeyCached, (version) => {
+                    if (version === matchingBackupInfo.version) {
+                        resolve();
+                    }
+                });
+            });
```

Comment:
If you're only expecting `KeyBackupDecryptionKeyCached` to be emitted once, you can also spell this:

```typescript
            const keyBackupIsCached = emitPromise(aliceClient, CryptoEvent.KeyBackupDecryptionKeyCached);
            // ...
            const version = await keyBackupIsCached;
            expect(version).toEqual(matchingBackupInfo.version);
```

... which I think is clearer.